### PR TITLE
First Look of client page

### PR DESCRIPTION
**Pull request by harshkasyap**
- npm install --save @angular/material@latest
- npm install --save @angular/animations@latest
- npm install --save hammers
- npm install --save @angular/flex-layout@latest
- npm install --save @angular/cdk

- Ng generate component lab
- Update style.scss

#6

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,21 +1,9 @@
...
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
-
+<div class ="collaborators" fxLayout="row">
+  <mat-list fxFlex fxLayout="row">
+    <mat-list-item *ngFor="let collaborator of collaborators">
+      <img mat-list-avatar src={{collaborator}} alt={{collaborator}}>
```
Comment (VarunDandekar) [inline src/app/app.component.html]: No point in keeping src and alt same. If src fails alt will also fail. Ideally, alt should have the name of collaborator.

File: src/app/lab/lab.component.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div>
+  <mat-toolbar color="primary">  
+    <span>!NL@B</span>
+  </mat-toolbar>
+
+  <mat-card>
+    <button mat-button>Fun</button>
```
Comment (VarunDandekar) [inline src/app/lab/lab.component.html]: why different treatment for Fun?

File: src/app/app.component.html (hunk 1 above)
Comment (harshkasyap) [reply src/app/app.component.html]: currently name isn't there for collaborator, once i will update collaborator with all the properties, i have kept this in mind to update.

File: src/app/lab/lab.component.html (hunk 2 above)
Comment (harshkasyap) [reply src/app/lab/lab.component.html]: This whole theme actually should be updated, would you pick this today.

**Pull request merged**